.github/workflows/ci.yaml

```diff
@@ -0,0 +1,70 @@
+name: CI
+
+on: [push, pull_request]
+
+env:
+  UNITY_SERIAL: ${{ secrets.UNITY_SERIAL }}
+  UNITY_PASSWORD: ${{ secrets.UNITY_PASSWORD }}
+  UNITY_EMAIL: ${{ secrets.UNITY_EMAIL }}
+
+jobs:
+
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        platform:
+          #- StandaloneWindows
+          #- StandaloneWindows64
+          - StandaloneLinux64
+          #- StandaloneOSX
+          - WebGL
+          #- iOS
+          - Android
+          #- PS4
+          #- XboxOne
+          #- tvOS
+          #- Switch
+          #- WSAPlayer
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2.3.3
+        with:
+          submodules: recursive
+
+      - name: Build
+        uses: game-ci/unity-builder@v2.0-aplha-5
+        with:
+          unityVersion: 2019.4.0f1 # for some reason it doesn't extract it automatically
+          projectPath: samples/unity-of-bugs
+          targetPlatform: ${{ matrix.platform }}
+
+      - name: Upload artifacts
+        uses: actions/upload-artifact@v1
+        with:
+          name: Build artifacts (${{ matrix.platform }})
+          path: build
+
+  test:
+    runs-on: ubuntu-latest
```

Comment:
I wonder if we can run tests on Mono and IL2CPP

Reply:
Not sure what that implies